tensorflow-core/tensorflow-core-api/src/gen/annotations/org/tensorflow/op/Ops.java

```diff
@@ -8097,6 +8156,45 @@ public While whileOp(Iterable<Operand<?>> input, ConcreteFunction cond, Concrete
     return While.create(scope, input, cond, body, options);
   }
 
+  /**
+   * Temporary op for experimenting with jax2tf.
+   *  DO NOT USE THIS OP. It has no backwards compatibility guarantees. It is also
+   *  very likely to change. This op will be used only in jax2tf under an
+   *  experimental flag.
+   *  <p>This is an experimental op to allow a smooth evolution of jax2tf towards
+   *  emitting and serializing MHLO directly from JAX. At the moment this op
+   *  carries a serialized MHLO module, therefore there are no backward-compatibility
+   *  guarantees, and should not be used for serialization.
+   *  Eventually, the op will carry a MHLO object, which will have
+   *  backwards-compatibility guarantees.
+   *  <p>The serialized module must return a tuple if and only if the Sout is an empty
+   *  list or a list with more than 1 elements. The length of Tout and Sout must
+   *  match. This op always returns a tuple of results, even if the module returns
+   *  a single result.
+   *  <p>The handling of dynamic shapes is work-in-progress. At the moment, the
+   *  JAX lowering for dynamic shapes will prepend one dimension parameter to the
+   *  serialized module for each dimension whose value must be passed in.
+   *  The &quot;args&quot; correspond to the non-dimension arguments. During compilation
+   *  we compute the values of the dimension arguments based on the static shapes of
+   *  the &quot;args&quot;. In order to do this, we encode for each dimension argument a
+   *  specification of how to compute its value, as a string, in the form
+   *  &quot;&lt;arg_idx&gt;.&lt;axis_idx&gt;&quot;.
+   *  E.g., the specification &quot;2.1&quot; denotes the value args[2].shape[1].
+   *
+   * @param args A list of {@code Tensor} with possibly different types to be passed as arguments
+   *  to the HLO module.
+   * @param module A serialized computation, a text representation of mlir.Module.
+   * @param Sout List of output tensor shapes.
+   * @param Tout List of output tensor data types.
+   * @param dimArgsSpec the specification for the dimension arguments, one for each
+   *  dimension argument. In absence of dynamic shapes this list is empty.
+   * @return a new instance of XlaCallModule
+   */
+  public XlaCallModule xlaCallModule(Iterable<Operand<?>> args, String module, List<Shape> Sout,
```

Comment:
Given the big warnings all over the docs for this op, should it be visible?

Reply:
Good catch!